Fix issues with missing jar files

kirpasingh opened the pull request:
Without building the root project (as suggested in the proposed edit), required dependencies cannot be found e.g. <b>dice-provider-emulator-1.0.0.jar</b>, thus the project fails to build.

azurecla commented:
Hi __@sjwaight__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

BryanLa commented:
Reviewing w/Markus

BryanLa commented:
Still waiting on review

BryanLa commented:
Third ping. Will close if no response.

Commit:
Fix issues with missing jar files

Without building the root project (as suggested in the proposed edit), required dependencies cannot be found e.g. <b>dice-provider-emulator-1.0.0.jar</b>, thus the project fails to build.
